make fixtures crashes on payment fixtures

### Describe the bug
Creating fixtures crashes on creating payment fixtures:

```bash
$ make fixtures
...
Creating a payment
Traceback (most recent call last):
  File "/home/joren/projects/concrexit/website/manage.py", line 19, in <module>
    execute_from_command_line(sys.argv)
  File "/home/joren/.cache/pypoetry/virtualenvs/concrexit-zE6WZ1Zu-py3.9/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/home/joren/.cache/pypoetry/virtualenvs/concrexit-zE6WZ1Zu-py3.9/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/joren/.cache/pypoetry/virtualenvs/concrexit-zE6WZ1Zu-py3.9/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/joren/.cache/pypoetry/virtualenvs/concrexit-zE6WZ1Zu-py3.9/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/home/joren/projects/concrexit/website/utils/management/commands/createfixtures.py", line 731, in handle
    self.create_payment()
  File "/home/joren/projects/concrexit/website/utils/management/commands/createfixtures.py", line 561, in create_payment
    create_payment(
  File "/home/joren/projects/concrexit/website/payments/services.py", line 45, in create_payment
    raise PaymentError(
payments.exceptions.PaymentError: User processing payment does not have the right permissions
make: *** [Makefile:55: fixtures] Error 1
```

### How to reproduce
Steps to reproduce the behaviour:
1. Remove your local database.
2. Run `make fixtures`

--- a/website/utils/management/commands/createfixtures.py
+++ b/website/utils/management/commands/createfixtures.py
@@ -557,10 +557,17 @@ class Command(BaseCommand):
 
         registration = event.registrations.order_by("?")[0]
 
-        processed_by = Member.objects.order_by("?")[0]
+        superusers = Member.objects.filter(is_superuser=True)
+        if not superusers:
+            print(
+                "There is no member which is also a superuser. Creating payments without this isn't possible!"
+            )
+            print("Please add an membership to the superuser.")
+            return
+
         create_payment(
             registration,
-            processed_by,
+            superusers[0],
             random.choice([Payment.CASH, Payment.CARD, Payment.WIRE]),
         )
 

